app/views/admin/events/_proposal.html.haml

```diff
@@ -83,6 +84,17 @@
                                             @event,
                                             event: { difficulty_level_id: difficulty.id }),
                 method: :patch
+      %tr
+        %td
+          %b Requires Registration
```

Comment:
No particular reason. I think it should be there and I should have added it long before.